src/modules/CRUD/components/Details/ExportGeomFiles/ExportGeomFiles.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Api from '@terralego/core/modules/Api';
+import { useTranslation } from 'react-i18next';
+import { Classes, MenuDivider, MenuItem } from '@blueprintjs/core';
+
+import { sanitizeCustomEndpoint } from '../../../services/utils';
+
+const ExportGeomFiles = ({ files }) => {
+  const list = Object.entries(files);
+  const { t } = useTranslation();
+
+  const count = list.length;
+
+  if (!count) {
+    return null;
+  }
+
+  const handleClick = async event => {
```

Comment:
useCallback maybe ?

Reply:
In this function, I don't manipulate any props or states, do you think it's necessary?